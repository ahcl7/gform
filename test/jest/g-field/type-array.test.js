import { generateComponent } from "./common";

describe('GField with type array', function () {
  it('should render correctly', () => {
    const component = generateComponent({
      key: 'addressArray1', type: 'array', label: 'Address1', flex: 'md-3',
      fields: [{ key: 'street', type: 'input', flex: 'md-6' }, { key: 'city', type: 'input', flex: 'md-6' },]
    });
    expect(component.html()).toMatchSnapshot();
  });
});
