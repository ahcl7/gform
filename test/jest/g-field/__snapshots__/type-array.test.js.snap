// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`GField with type array should render correctly 1`] = `
<g-col-stub xs12
            in-array="false"
>
  <g-btn-stub class="ma-2"
              textcolor="blue lighten-2"
              outlined
              small
  >
    ADD ADDRESS1
  </g-btn-stub>
</g-col-stub>
`;
